sdk/storage/azure_storage_blob/src/lib.rs

```diff
@@ -11,66 +11,67 @@ pub mod clients {
     pub mod blob_container_client;
     pub mod blob_service_client;
 
-    pub use blob_client::BlobClient;
-    pub use blob_container_client::BlobContainerClient as ContainerClient;
-    pub use blob_service_client::BlobServiceClient as ServiceClient;
-
     pub use crate::generated::clients::{
-        BlobAppendBlobClient, BlobBlobClient, BlobBlockBlobClient,
-        BlobClient as GeneratedBlobClient, BlobClientOptions, BlobContainerClient,
-        BlobPageBlobClient, BlobServiceClient,
+        AppendBlobClient, AppendBlobClientOptions, BlobClient as GeneratedBlobClient,
+        BlobClientOptions, BlobContainerClient as GeneratedBlobContainerClient,
+        BlobContainerClientOptions, BlobServiceClient as GeneratedBlobServiceClient,
+        BlobServiceClientOptions, BlockBlobClient, BlockBlobClientOptions, PageBlobClient,
+        PageBlobClientOptions,
     };
+    pub use blob_client::BlobClient;
+    pub use blob_container_client::BlobContainerClient;
+    pub use blob_service_client::BlobServiceClient;
```

Comment:
We still have a collision on the generated client names and our handwritten client names, not sure if we want to actually resolve this or if renaming the export at the `lib.rs` level is idiomatic and suffices.